package.json

```diff
@@ -18,5 +18,8 @@
   },
   "engines": {
     "node": "^16.18"
+  },
+  "dependencies": {
+    "express-session": "^1.17.3"
```

Comment:
nope. first, this was added to your root package.json, which is wrong -> you needto add it to server/package.json

second, we dont want you toying with the core dependencies of this app. the downgrade of passport should have been enough!